.circleci/config.yml

```diff
@@ -89,6 +89,22 @@ commands:
       - run: git submodule sync
       - run: git submodule update --init
 
+  skip-if-docs-only:
+    steps:
+      - run: git remote add upstream https://github.com/determined-ai/determined
+      - run: git fetch upstream
+      - run:
+          name: check for docs-only changes
+          command: |
+            MERGE_BASE="$(git merge-base upstream/master HEAD)"
+            DIFF_DIRS="$(git diff-tree --no-commit-id --name-only "$MERGE_BASE" HEAD)"
+            if [ "$DIFF_DIRS" = ".circleci" ] ; then
```

Comment:
Before landing, this should be `"docs"` not `".circleci"`, but setting it to .circleci makes it easy for us to test the docs-only behavior in this PR